src/graccreq/oim/OIMTopology.py

```diff
@@ -104,7 +110,7 @@ def get_file_from_OIM():
         try:
             oim_xml = urllib2.urlopen(oim_url)
         except (urllib2.HTTPError, urllib2.URLError) as e:
-            print e
+            logging.warn(e)
```

Comment:
I thought about it some more, and I think it's probably better as an error. The way the logic works right now, we only get to this point if the cache file is ruled invalid (too old or doesn't exist).  So if this fails, the records won't get filled in, which as you said, should really be an error.

I'll make that change now.